circeSupport/src/test/scala/CirceSupportSpec.scala

```diff
@@ -0,0 +1,48 @@
+import io.buildo.enumero._
+import io.buildo.enumero.circe._
```

Comment:
Why do we need to `import io.buildo.enumero.circe._` ?

Reply:
we actually only use `DecodingFailure` and `Json`, I'll make it explicit 👍